ansible/roles/jenkins-worker/tasks/main.yml

```diff
@@ -160,6 +173,32 @@
   command: systemctl restart firewalld
   when: "'fedora27-x64' in inventory_hostname"
 
+- name: Check if SU is installed on IBMi
+  when: os|startswith("ibmi")
+  stat:
+    path: /QOpenSys/pkgs/bin/su
+  register: su_exists
+
+- name: git clone su on ibmi
```

Comment:
better to be a `curl https://....` command. we can host .tgz files in our own internal ci infra, curl it, an untgz it and not have deps on external resources. Some manual setup info on how the `su` was produced so it can be reproduced if necessary would be helpful, too.